devicetypes/zooz/zooz-zse44-temperature-humidity-xs-sensor.src/zooz-zse44-temperature-humidity-xs-sensor.groovy

```diff
@@ -0,0 +1,424 @@
+@Field static Map humiditySensor = [sensorType: 5, scale:0]
+@Field static Map temperatureAlarm = [name:"temperatureAlarm", notificationType:4, normalEvent:0, highEvent:2, lowEvent:6, highValue:"heat", lowValue:"freeze", normalValue:"cleared"]
+@Field static Map humidityAlarm = [name:"humidityAlarm", notificationType:16, normalEvent:0, highEvent:2, lowEvent:6, highValue:"high", lowValue:"low", normalValue:"normal"]
+@Field static int wakeUpInterval = 43200
+
+metadata {
+	definition (
+		name: "Zooz ZSE44 Temperature | Humidity XS Sensor",
+		namespace: "Zooz",
+		author: "Kevin LaFramboise (@krlaframboise)",
+		ocfDeviceType:"oic.d.thermostat",
+		vid: "b68c78d7-bd01-3717-a2ac-d1d55ce5ef73",
+		mnmn: "SmartThingsCommunity"
+	) {
+		capability "Sensor"
+		capability "Temperature Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Battery"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Configuration"
+		capability "platemusic11009.temperatureHumiditySensor"
+		capability "temperatureAlarm"
+		capability "platemusic11009.humidityAlarm"
+		capability "platemusic11009.firmware"
+		capability "platemusic11009.syncStatus"
+
+		// zw:Ss2a type:0701 mfr:027A prod:7000 model:E004 ver:1.10 zwv:7.13 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,31,72,5A,87,73,80,71,70,84,7A
+		fingerprint mfr:"027A", prod:"7000", model:"E004", deviceJoinName: "Zooz Multipurpose Sensor" // Zooz ZSE44 Temperature | Humidity XS Sensor
+	}
+
+	preferences {
+		configParams.each { name, param ->
+			if (param.options) {
+				input name, "enum",
+					title: param.title,
+					description: "Default: ${param.options[param.defaultVal]}",
+					required: false,
+					displayDuringSetup: false,
+					defaultValue: param.defaultVal,
+					options: param.options
+			} else if (param.range) {
+				input name, "number",
+					title: param.title,
+					description: "${param.desc} - Default: ${param.defaultVal}",
+					required: false,
+					displayDuringSetup: false,
+					defaultValue: param.defaultVal,
+					range: param.range
+			}
+		}
+
+		input "debugLogging", "enum",
+			title: "Logging:",
+			description: "Default: Enabled",
+			required: false,
+			defaultValue: "1",
+			options: ["0":"Disabled", "1":"Enabled"]
+	}
+}
+
+def installed() {
+	logDebug "installed()..."
+	state.pendingRefresh = true
+	initialize()
+}
+
+def updated() {
+	logDebug "updated()..."
+	initialize()
+
+	if (pendingChanges) {
+		logForceWakeupMessage("The setting changes will be sent to the device the next time it wakes up.")
+	}
+}
+
+void initialize() {
+	state.debugLoggingEnabled = (safeToInt(settings?.debugLogging, 1) != 0)
+
+	refreshSyncStatus()
+
+	if (device.currentValue("temperatureHumidity") == null) {
+		state.displayHumidity = " "
+		state.displayTemperature = " "
+		sendEvent(name:"temperatureHumidity", value:" ")
+	}
+
+	if (!device.currentValue("temperatureAlarm")) {
+		sendEvent(name:"temperatureAlarm", value:temperatureAlarm.normal)
```

Comment:
I don't think `temperatureAlarm.normal` is defined?

Reply:
That value is coming from a static map on line 71...

@Field static Map temperatureAlarm = [name:"temperatureAlarm", notificationType:4, normalEvent:0, highEvent:2, lowEvent:6, highValue:"heat", lowValue:"freeze", normalValue:"cleared"]

Reply:
Nice catch, I've made the change.